8:15 updated IG to match new spec

Abhijay opened the pull request.

Abhijay committed:
8:15 updated IG to match new spec

cmoesel commented on fhirG5/CodeSystem-shr-actor-LanguageQualifierCS.html:
```diff
@@ -1,168 +1,172 @@
+---
+redirect_from:
+  - /fhir/CodeSystem/shr-actor-LanguageQualifierCS/
+---
```
What is this redirect stuff at the top of the file?  That's not in the previous version.

Abhijay committed:
removed ambiguous redirects

fhirG5 links were being linked to with /fhir redirects, which were also being used by the original grammar 4 files. As such, the ambiguous redirects were removed.

Abhijay committed:
updated again after updating yarn

Abhijay committed:
solved -tx issue, as well as added spec

also, the spec/shr/path/to/individual/html/files was no longer automatically included in the fhir output, so I added those in (e.g /fhirG5/spec/shr/actor/EmergencyContact.html)
